common/flatpak-dir.c

```diff
@@ -11995,22 +12042,32 @@ sort_strv (char **strv)
 char **
 flatpak_dir_get_default_locale_languages (FlatpakDir *self)
 {
-  char **langs = NULL;
+  g_autoptr(GPtrArray) langs = g_ptr_array_new ();
```

Comment:
I think this should be `g_ptr_array_new_with_free_func (g_free)` to properly document the ownership.